plugin/sampling/strategystore/adaptive/options.go

```diff
@@ -0,0 +1,153 @@
+	TargetQPS float64
+
+	// QPSEquivalenceThreshold is the acceptable amount of deviation for the operation QPS from the `targetQPS`,
+	// ie. [targetQPS-equivalenceThreshold, targetQPS+equivalenceThreshold] is the acceptable targetQPS range.
+	// Increase this to reduce the amount of fluctuation in the probability calculation.
+	QPSEquivalenceThreshold float64
+
+	// LookbackQPSCount determines how many previous operation QPS are used in calculating the weighted QPS,
+	// ie. if LookbackQPSCount is 1, the only the most recent QPS will be used in calculating the weighted QPS.
+	LookbackQPSCount int
+
+	// CalculationInterval determines the interval each bucket represents, ie. if an interval is
+	// 1 minute, the bucket will contain 1 minute of throughput data for all services.
+	CalculationInterval time.Duration
+
+	// LookbackInterval is the total amount of throughput data used to calculate probabilities.
+	LookbackInterval time.Duration
+
+	// Delay is the amount of time to delay probability generation by, ie. if the calculationInterval
+	// is 1 minute, the number of buckets is 10, and the delay is 2 minutes, then at one time
+	// we'll have [now()-12,now()-2] range of throughput data in memory to base the calculations
+	// off of.
+	Delay time.Duration
+
+	// DefaultSamplingProbability is the initial sampling probability for all new operations.
+	DefaultSamplingProbability float64
+
+	// MinSamplingProbability is the minimum sampling probability for all operations. ie. the calculated sampling
+	// probability will be bound [MinSamplingProbability, 1.0]
+	MinSamplingProbability float64
+
+	// LowerBoundTracesPerSecond determines the lower bound number of traces that are sampled per second.
+	// For example, if the value is 0.01666666666 (one every minute), then the sampling processor will do
+	// its best to sample at least one trace a minute for an operation. This is useful for a low QPS operation
+	// that is never sampled by the probabilistic sampler and depends on some time based element.
+	LowerBoundTracesPerSecond float64
+
+	// LeaderLeaseRefreshInterval is the duration to sleep if this processor is elected leader before
+	// attempting to renew the lease on the leader lock. NB. This should be less than FollowerLeaseRefreshInterval
+	// to reduce lock thrashing.
+	LeaderLeaseRefreshInterval time.Duration
+
+	// FollowerLeaseRefreshInterval is the duration to sleep if this processor is a follower
+	// (ie. failed to gain the leader lock).
+	FollowerLeaseRefreshInterval time.Duration
+}
+
+// AddFlags adds flags for Options
+func AddFlags(flagSet *flag.FlagSet) {
+	flagSet.Float64(targetQPS, defaultTargetQPS,
+		"The target number of sampled traces for all operations.",
+	)
+	flagSet.Float64(equivalenceThreshold, defaultEquivalenceThreshold,
+		"The acceptable amount of deviation for the operation QPS from the `targetQPS`. Increase this to reduce the amount of fluctuation in the probability calculation.",
```

Comment:
acceptable for what? If the actual delta is smaller, then ... ?  No update to the probability?